packages/flutter/lib/src/material/chip_theme.dart

```diff
@@ -326,6 +328,19 @@ class ChipThemeData extends Diagnosticable {
   ///  * [shadowColor]
   final Color selectedShadowColor;
 
+  /// Whether or not to show a check mark when [selected] is true.
+  ///
+  /// For instance, the [ChoiceChip] sets this to false so that it can be
+  /// be selected without showing the check mark.
+  ///
+  /// Defaults to true.
+  final bool showCheckmark;
```

Comment:
Ah I think I misunderstood above (when I said this should default to true).

Would you consider changing the name to `showCheckmarkWhenSelected`? Or is that too wordy?

Reply:
It already existed as showCheckmark, I just moved it to Checkmarkable interface